public/js/admin.js

```diff
@@ -47,6 +47,11 @@
     $("#messageDisplay").show().one();
   });
 
+  socket.on('expired token', function(service){
+    alert('Your session has expired. Please log in again.');
+    window.location.replace('https://cas.byu.edu/cas/logout');
+    window.location.replace('https://cas.byu.edu/cas/login?service=' + service);
```

Comment:
No. I couldn't figure out a way for it to redirect to the logout page and then to go to the login page. However, it does update the token when they re-login.